mm2src/coins/z_coin.rs

```diff
@@ -1153,15 +1202,44 @@ impl MarketCoinOps for ZCoin {
         ))
     }
 
+    /// Calculates the wallet balance, divided into spendable and unspendable portions.
+    /// Unspendable balance consists of notes that are locked in the wallet.
     fn my_balance(&self) -> BalanceFut<CoinBalance> {
         let coin = self.clone();
         let fut = async move {
-            let sat = coin
+            let spendabled_notes = coin
+                .get_spendable_notes()
+                .await
+                .map_err(|err| BalanceError::WalletStorageError(err.to_string()))?;
```

Comment:
I don't think the function and the variable name make much sense after changes in this PR, please think of an appropriate name, `get_available_notes` is one :)
P.S. AI suggested `get_wallet_notes`

Reply:
done

Reply:
I suggested renaming the function as well :)

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2331/commits/34efbde817b0b0de98a84158f0d7564c70c149a2